core/src/main/java/io/fundrequest/core/request/claim/ClaimServiceImpl.java

```diff
@@ -23,15 +25,18 @@
     private RequestClaimRepository requestClaimRepository;
     private GithubClaimResolver githubClaimResolver;
     private Mappers mappers;
+    private ApplicationEventPublisher $;
```

Comment:
This variable should have a more descriptieve name (e.g. `private ApplicationEventPublisher eventPublisher`)